ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/AbstractBlockCreator.java

```diff
@@ -224,6 +228,10 @@ protected BlockCreationResult createBlock(
       operationTracer.traceStartBlock(
           disposableWorldState, processableBlockHeader, miningBeneficiary);
 
+      final BlockAccessListBuilder blockAccessListBuilder = new BlockAccessListBuilder();
+      final Optional<TransactionAccessList> preExecutionAccessList =
```

Comment:
as we will do in the PR https://github.com/hyperledger/besu/actions/runs/17827720658?pr=9189
we should only activate it if it's in the fork

Reply:
Done